rules/scripts.mk

```diff
@@ -23,11 +23,15 @@ $(SYSCTL_NET_CONFIG)_PATH = files/image_config/sysctl
 UPDATE_CHASSISDB_CONFIG_SCRIPT = update_chassisdb_config
 $(UPDATE_CHASSISDB_CONFIG_SCRIPT)_PATH = files/scripts
 
+SWSS_SYNCD_VARS_TEMPLATE = swss_syncd_vars.j2
```

Comment:
>SWSS_SYNCD_VARS_TEMPLATE [](start = 0, length = 24)

Wondering why original code don't need it.

Reply:
It is needed here to make the file available to be copied to dvs and p4 docker. The template file was not available in the platforms.